frame/election-provider-support/solution-type/src/single_page.rs

```diff
@@ -178,6 +179,25 @@ pub(crate) fn generate(def: crate::SolutionDef) -> Result<TokenStream2> {
 			<#ident as _feps::NposSolution>::TargetIndex,
 			<#ident as _feps::NposSolution>::Accuracy,
 		>;
+		impl _feps::codec::MaxEncodedLen for #ident {
+			fn max_encoded_len() -> usize {
+				use frame_support::traits::Get;
+				use _feps::codec::Encode;
+				let s: u32 = #max_voters::get();
+				// The last element of the struct is a vec with 1 voter
+				// then #count-1 tuple of target with an accuracy
+				// and then lastly the final target
+				let max_element_size = #voter_type::max_encoded_len()
+					.saturating_add((#count - 1)
+						.saturating_mul(#target_type::max_encoded_len()
+							.saturating_add(#weight_type::max_encoded_len())))
+					.saturating_add(#target_type::max_encoded_len());
```

Comment:
is this just github, or is the indentation here weird?

Reply:
Changed as per @kianenigma feedback below.